src/validator.js

```diff
@@ -0,0 +1,19 @@
+import Ajv from "ajv";
+
+const ajv = new Ajv();
+const schema = {
+  "properties": {
+    "url": {
+      "type": "string",
+      "format": "url"
```

Comment:
Use `uri` instead of `url`. `url` is an AJV extension to JSON Schema and it doesn't fit our needs. Most importantly, it doesn't work for localhost URLs. Ideally we would use `iri`, but AJV doesn't support that format. http://json-schema.org/latest/json-schema-validation.html#rfc.section.7.3.5

Reply:
Done